packages/catalog-search/src/tests/SearchSuggestionItem.test.jsx

```diff
@@ -0,0 +1,115 @@
+import { renderWithRouter } from '@edx/frontend-enterprise-utils';
+import React from 'react';
+import { screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import SearchSuggestionItem from '../SearchSuggestionItem';
+
+describe('<SeachSuggestionItem />', () => {
+  test('renders course data', () => {
+    const mockData = {
+      course: {
+        url: '/test-enterprise/course/edX+courseX',
+        suggestionItemHandler: jest.fn(),
+        disableSuggestionRedirect: true,
+        hit: {
+          content_type: 'course',
+          key: 'edX+courseX',
+          title: 'test-course',
+          _highlightResult: { title: { value: '<em>course</em> <em>catalog</em>' } },
+        },
+      },
+    };
+
+    renderWithRouter(<SearchSuggestionItem
+      url={mockData.course.url}
+      suggestionItemHandler={mockData.course.suggestionItemHandler}
+      hit={mockData.course.hit}
+      disableSuggestionRedirect={mockData.course.disableSuggestionRedirect}
+    />);
+    expect(screen.getByRole('link', { name: 'course catalog edX' })).not.toBeNull();
+    expect(screen.getByText('course')).not.toBeNull();
+    expect(screen.getByText('catalog')).not.toBeNull();
+    expect(screen.getByText('edX')).not.toBeNull();
```

Comment:
which edX text is this finding?

Reply:
is it the course key? If so then maybe assert the whole key

Reply:
it's testing for the badge text so I believe it should only be `edX`